website/src/components/CRONComponents/Chart/Chart.jsx

```diff
@@ -0,0 +1,109 @@
+/*
+Copyright 2023 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from "react";
+import { ResponsiveLine } from "@nivo/line";
+
+const ResponsiveChart = ({ data, title, colors, isFirstChart }) => {
+  return (
+    <div className="chart">
+      {data.length > 0 && (
```

Comment:
Should we return early from `ResponsiveChart` in case `data.length` is equal to zero? That would prevent us from creating an empty div